morpheus-apollo-1/genesis.json

```
@@ -10,7 +10,7 @@
     },
     "evidence": {
       "max_age_num_blocks": "100000",
-      "max_age_duration": "172800000000000",
+      "max_age_duration": "259200s",
```

```suggestion
      "max_age_duration": "259200",
```
The `s` will invalidate the genesis

---

@bragaz Your suggestion would cause the `max_age_duration` to change from 259.200 seconds (3 days) to 259.200 nanoseconds (~ 0.000000002 seconds). 

The correct change is to multiply the seconds value by 1.000.000.000, which represents the amount of nanoseconds per second:

```suggestion
      "max_age_duration": "259200000000000",
```

---

Whoops! Forgot to think about zeroes!